backend/pick-git/src/test/java/com/woowacourse/pickgit/acceptance/post/PostAcceptanceTest.java

```diff
@@ -492,6 +496,180 @@ void showRepositories_InvalidUsername_400Exception() {
         assertThat(response.getErrorCode()).isEqualTo("V0001");
     }
 
+    @DisplayName("로그인 사용자는 게시물을 좋아요 할 수 있다. - 성공")
+    @Test
+    void likePost_LoginUser_Success() {
+        // given
+        String loginUserToken = 로그인_되어있음(USERNAME).getToken();
+        String targetUserToken = 로그인_되어있음(ANOTHER_USERNAME).getToken();
+        Long postId = 1L;
+
+        requestToWritePostApi(targetUserToken, HttpStatus.CREATED);
+
+        // when
+        LikeResponse response = given().log().all()
+            .auth().oauth2(loginUserToken)
+            .when()
+            .put("/api/posts/" + postId + "/likes")
```

Comment:
```suggestion
            .put("/api/posts/{postId}/likes", postId)
```
테스트의, 살아있는 문서라는 관점을 잘 살리는데, 이와같은 표기법이 더 가독성이 좋다고 생각하는데 어떻게 생각하세요